omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -129,11 +139,39 @@
 				
 				<prop key="admin/provider/providerAttributeType.list">providerAttributeTypeList</prop>
 				<prop key="admin/provider/providerAttributeType.form">providerAttributeTypeForm</prop>	
-				
+
  			</props>
 		</property>
 	</bean>	
 
+ 	<!-- index pages - begin -->  
+	
+	<!-- admin -->
+	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
```

Comment:
@jdegraft There are probably a few ways you can map URLs like /admin without making changes to the JSPs (btw if you do change links in the JSP they should reference /admin/index.htm). 
If you were to go back to the annotation method and re-introduce AdminPageController, you could write another method with @RequestMapping(value = "/admin") that returns "module/legacyui/admin/index".

Another thing you could do would be to put a mapping for /admin into the legacyUiUrlMapping bean and map it to an appropriate controller. The UrlFilenameViewController has a handy prefix and a suffix property, so if you were to set the prefix to "/module/legacyui" and the suffix to "/index", it looks like it would map a URL of /admin to /module/legacyui/admin/index. You could reuse that same controller instance in the legacyUiUrlMapping bean from /admin, /patients, etc, anything that has an 'index.jsp' underneath.

BTW r.e. view resolution -- the only reason /admin maps to /WEB-INF/view/admin.jsp right now is because there is no other explicit controller mapping in Spring, so Spring falls back to the 'springController' bean which is a UrlFilenameViewController (without a prefix or suffix) which interprets paths like /admin as a view named "admin", which then gets looked up by the internal view resolver as /WEB-INF/view/<viewname>.jsp. As soon as you add a mapping like /admin to another url handler mapping, it should work...